File: solutions/csharp/cricinfo/Player.cs
namespace Cricinfo
{
    public class Player
    {
        private readonly string id;
        private readonly string name;
        private readonly string role;

        public Player(string id, string name, string role)
        {
            this.id = id;
            this.name = name;
            this.role = role;
        }
    }
}

File: solutions/python/cricinfo/player.py
from player_stats import PlayerStats

class Player:
    def __init__(self, player_id, name, role):
        self.id = player_id
        self.name = name
        self.role = role
        self.stats = PlayerStats()

    def get_id(self):
        return self.id

    def get_name(self):
        return self.name

    def get_role(self):
        return self.role

    def get_stats(self):
        return self.stats